scripts/spack/configs/linux_ubuntu_24/spack.yaml

```diff
@@ -1,36 +1,21 @@
-# Copyright (c) 2017-2025, Lawrence Livermore National Security, LLC and
+# Copyright (c) 2017-2023, Lawrence Livermore National Security, LLC and
```

Comment:
Is there a reason the copyright end date reverted from 2025 to 2023?

Reply:
Good catch, copy paste error here